src/braket/experimental/autoqasm/instructions/pulse_control.py

```diff
@@ -0,0 +1,152 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Pulse control instructions that apply to frames or qubits.
```

Comment:
Is this file called `pulse_control` because there's already a file called `pulse`?

Reply:
No yet, but I plan to reserve the `aq.pulse` for the decorator.

Reply:
Also, the instructions here are to control/compose a pulse rather than the pulse itself.

Reply:
From the examples in the tests, it confuses me why we need a pulse decorator. We can intermix them as much as we want. If the decorator is to designate a pulse gate, it should be called `aq.defcal`, not `pulse`.

I like `pulse` because it is often used to describe this level of programming, so a `set_freq` instruction is a "pulse" instruction, even if it's not playing a waveform